_includes/head.html

```diff
@@ -13,4 +13,8 @@
   {% if jekyll.environment == 'production' and site.google_analytics %}
   {% include google-analytics.html %}
   {% endif %}
+
+  {% if site.gems contains 'jekyll-seo-tag' %}
+  {% include seo.html %}
```

Comment:
> But that doesn't explain the need of having {% seo title=false %} by default.

@ashmaroli no particular reason, except that it was the minimal implementation since title is already being defined above.